packages/@lwc/compiler/src/compiler/__tests__/result.spec.ts

```diff
@@ -245,7 +245,7 @@ export default class Test extends LightningElement {
 
             expect(mainDefMappedToOutputPosition).toMatchObject({
                 line: 14,
-                column: 15,
+                column: 11,
```

Comment:
The generated sourcemap in **master branch** is wrong, this is the correct position on the source code.